features/signup_to_account.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Signup to an account
+  As a new user 
+  In order to be able to access the app 
+  I should be able to signup to an account
+
+  Scenario: As a user I would like to signup to an account
+    Given I am on the landing page
+    When I click on the "Sign up" button
+    Then I fill in "Name" with "Bob"
```

Comment:
Wrong usage of the Given When Then steps. Read this blog post to get a better understanding https://elabor8.com.au/a-practical-blog-on-how-to-write-scenarios-using-bdd/